This file is part of mingw-cross-env.
See doc/index.html for further information.

This patch has been taken from:
http://permalink.gmane.org/gmane.text.xml.soap.gsoap/1933

diff -urN a/gsoap/stdsoap2.h b/gsoap/stdsoap2.h
--- a/gsoap/stdsoap2.h	2010-09-19 01:20:12.000000000 +0200
+++ b/gsoap/stdsoap2.h	2010-09-19 21:00:31.442187011 +0200
@@ -536,6 +536,7 @@
 
 #ifdef WITH_C_LOCALE
 # include <xlocale.h>
+# include <locale.h>
 #else
 # undef HAVE_STRTOF_L
 # undef HAVE_STRTOD_L
